setup.cfg

```diff
@@ -0,0 +1,11 @@
+[flake8]
```

Comment:
If you expand this file to add the metadata and options from the daf_butler file you should be able to pip install this package and then run tests.